doc/news/changes/minor/20190817PeterMunch

```diff
@@ -0,0 +1,4 @@
+New: Introduce new class parallel::DistributedTriangulationBase 
+between parallel::Triangulation and parallel::distributed::Triangulation.
```

Comment:
```suggestion
between parallel::TriangulationBase and parallel::distributed::Triangulation.
```

Reply:
You are right. This is the old documentation...